app/views/search/index.html.erb

```diff
@@ -0,0 +1,17 @@
+<p>Closest Fuel Station Near Here:</p>
+<% @station.each do |l| %>
```

Comment:
It's bad practice to name your block variables single letters, and to name a collection singularly. This should be `@stations.each do |station|`